### DISCUSSION TITLE
aax audible converter

### DISCUSSION THREAD

**Discussion by jainca** [audiamus/AaxAudioConverter#148] [Q&A]
I have the Audible Manager installed and working on my Win 10 PC. I installed the aax audible converter so that I can convert my purchased Audible book to MP3 format. I no longer have membership to Audible but many purchased books that I want to convert to MP3 format. However, the AAX Converter is all greyed out and I am not able to do anything with it. 
Please note that I am NOT an IT person. Can someone tell me what to do. Many many thanks.

**Comment by audiamus** [comment]
By "Audible Manager" you mean the legacy Windows app with its own integrated player? 

As far as I understood Audible policies you should still have access to your books - you "purchased" them (by Audible terms) - and you should be able to download them with Audible Manager. You will need to log-in, though. Your Audible account should still exist, even if you cancelled your subscription. Connecting Audible Manager to your Audible account will also download your key for your aax files ("activation bytes") in the background. AAX Audio Converter should then be able to find the key and apply it.

Alternatively, you can try to use [Book Lib Connect](https://github.com/audiamus/BookLibConnect) for downloading, but again, you will have to connect to your Audible or Amazon account.
